style.css

```diff
@@ -120,7 +122,8 @@ p {
 }
 
 .lang {
-  display: flex;
+  display: grid;
+  grid-template-columns: auto auto;
   justify-content: space-between;
   margin: 0  15px 0 15px;
   color: #fff;
```

Comment:
Kindly display your .lang class to flex not grid. you should only display your main container for the cards to grid and the main container should be a section tag not a div tag change the div tag in line 41 in your html file to a section tag and display it to grid